app/views/shared/_search.html.erb

```diff
@@ -65,6 +65,17 @@
           </fieldset>
         </div>
         <div>
+          <% if @search_form.application_type == 'crm7' %>
```

Comment:
This isn't quite the right check. On the dashboard, `application_type` is crm7 if the user has previously chosen the NSM radio button and submitted the form. So you'll get the weird behaviour that at first this field will be hidden, then if you choose NSM it will appear after you first submit the form. Whereas the ticket says that it should always be visible on the dashboard search form. You probably want `<% if choose_application_type || @search_form.application_type == 'crm7' %>`